Userland/Libraries/LibCore/FileSystem.cpp

```diff
@@ -231,15 +240,16 @@ ErrorOr<void, CopyError> copy_file(DeprecatedString const& dst_path, DeprecatedS
             src_stat.st_mtim,
 #endif
         };
-        if (utimensat(AT_FDCWD, dst_path.characters(), times, 0) < 0)
+        DeprecatedString dst_path_string = dst_path;
+        if (utimensat(AT_FDCWD, dst_path_string.characters(), times, 0) < 0)
             return CopyError { errno, false };
     }
     return {};
 }
 
-ErrorOr<void, CopyError> copy_directory(DeprecatedString const& dst_path, DeprecatedString const& src_path, struct stat const& src_stat, LinkMode link, PreserveMode preserve_mode)
+ErrorOr<void, CopyError> copy_directory(StringView dst_path, StringView src_path, struct stat const& src_stat, LinkMode link, PreserveMode preserve_mode)
 {
-    if (mkdir(dst_path.characters(), 0755) < 0)
+    if (System::mkdir(dst_path, 0755).is_error())
```

Comment:
Other places use `Core::System` instead of just `System`, so we should probably keep that as consistent as possible.

Reply:
IMO, it's fine not to when we're already in the namespace, but we're a bit inconsistent throughout the codebase of whether we do or don't.

Reply:
I've made it consistently use `System::` as that seemed reasonable. If it needs to be the `Core::System` I'm happy to do that to!

Reply:
I meant to express that we should pick one of both ("other places" being other places in the same file) and use that consistently, so this is fine. :^)